lua/octo/completion.lua

```diff
@@ -48,38 +49,51 @@ local function get_repo_issues(repo, params)
 end
 
 function M.issue_complete(findstart, base)
-  -- :he complete-functions
+  -- the complete-functions
```

Comment:
:) that was vim `:help` command, but its ok wither way :)

Reply:
Ah! I always just use `:h` so I didn't recognize it. I'll change it to `:help complete-functions`

Reply:
This should be resolved [here](https://github.com/pwntester/octo.nvim/pull/48/commits/f6771a23b5f6126baaba6a7e7312daaa14127e7b)